src/test/kotlin/org/rust/ide/docs/RsQualifiedNameProviderTest.kt

```diff
@@ -0,0 +1,80 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.docs
+
+import com.intellij.ide.actions.CopyReferenceAction
+import org.intellij.lang.annotations.Language
+import org.rust.InlineFile
+import org.rust.RsTestBase
+import org.rust.lang.core.psi.ext.RsReferenceElement
+import org.rust.lang.core.psi.ext.*
+import kotlin.test.assertEquals
+
+class RsQualifiedNameProviderTest : RsTestBase() {
```

Comment:
It would be great to add tests for combination of `Copy Reference` and `Paste` actions